index.html

```diff
@@ -6,18 +6,73 @@
     <meta name="description" content="Your description goes here">
     <meta name="keywords" content="one, two, three">
 
-    <title>Blank Template</title>
+    <title>Relaxr</title>
 
     <!-- external CSS link -->
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,600,800" rel="stylesheet">
 
   </head>
 
   <body>
-    <div class="container">
-      <!-- Replace me with your html -->
-    </div>
+
+    <header>
+      <div class="container">
```

Comment:
I LOVE that you used the container INSIDE of those sections. That's how I would have done it!